pkg/controller/kubernetes/controller/nodelifecycle/config/types.go

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
please import `types.go` from `k8s.io/kube-controller-manager`

Reply:
The `k8s.io/controller-manager` only supports version 1.20.x or later. The current policy is to cache the used `k8s.io/kubernetes/pkg/controller` files. After openYurt is upgraded to version 1.20.x, I will submit a PR to replace `k8s.io/kubernetes/pkg/controller` with `k8s.io/controller-manager`.